app.py

```diff
@@ -23,6 +24,9 @@
 app.url_map.converters["regex"] = helpers.RegexConverter
 talisker.flask.register(app)
 
+if not app.testing:
```

Comment:
Why if not testing? Wouldn't we want to test with the same requests library as we use in production?